hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/PipelineStateManager.java

```diff
@@ -45,8 +45,7 @@ void addPipeline(HddsProtos.Pipeline pipelineProto)
 
   /**
    * Removing pipeline would be replicated to Ratis.
-   * @param pipelineIDProto
-   * @return Pipeline removed
+   * @param pipelineIDProto Pipeline
```

Comment:
Maube it would be better to remove `Pipeline`. Other methods have only `* @param pipelineIDProto`, without it.

Reply:
Done